docs/config-app.md

```diff
@@ -33,6 +33,7 @@ This parameter affects how many CPU cores will be utilized by the application. R
 - `http-client.max-redirects` - set the maximum amount of HTTP redirections to follow. A value of 0 (the default) prevents redirections from being followed.
 
 ## Auction (OpenRTB)
+- `auction.blacklisted-accts` - comma separated list of blacklisted account IDs.
```

Comment:
Pls rename to `auction.blacklisted-accounts`. Also, check all remaining variables/constant name to be human readable.